### DISCUSSION TITLE
About multiple @apply

### DISCUSSION THREAD

**Discussion by jet10000** [tailwindlabs/tailwindcss#4949] [Ideas]
```
body {
    @apply text-green-600
    @apply bg-green-100
}
```
 
I want to separate the styles better

**Comment by dance2die** [comment]
You can already use multiple `@apply`.

Demo: https://play.tailwindcss.com/gYLcUAflTq?file=css

If it's not working, it could be due to the missing semicolon.

```diff
body {
-  @apply text-green-600
+  @apply text-green-600;
  @apply bg-green-100;
}
```

**Reply by jet10000** [reply]
Thanks a lot
